package.json

```diff
@@ -8,22 +8,20 @@
   },
   "scripts": {
     "prepare": "husky install && yarn build",
-    "clean": "lerna run clean && lerna clean --yes && rm -rf node_modules",
     "lint": "eslint packages/ scripts/ --ext=js,mjs,ts",
     "format": "prettier --write '{packages,scripts}/**/*.{js,mjs,ts,json,md}'",
     "bundlesize": "bundlesize --config ./scripts/bundlesize/bundlesize.config.json",
-    "build": "lerna run build --ignore @lwc/perf-benchmarks --ignore @lwc/perf-benchmarks-components --ignore @lwc/integration-tests",
-    "build:performance": "yarn run build:performance:components && yarn run build:performance:benchmarks",
-    "build:performance:components": "lerna exec --scope @lwc/perf-benchmarks-components -- yarn build",
-    "build:performance:benchmarks": "lerna exec --scope @lwc/perf-benchmarks -- yarn build",
-    "dev": "lerna run dev --parallel --ignore @lwc/perf-benchmarks --ignore @lwc/perf-benchmarks-components --ignore @lwc/integration-tests",
+    "build": "nx run-many --target=build --all --exclude=@lwc/perf-benchmarks,@lwc/perf-benchmarks-components,@lwc/integration-tests",
+    "build:performance": "yarn build:performance:components && yarn build:performance:benchmarks",
+    "build:performance:components": "nx build @lwc/perf-benchmarks-components",
+    "build:performance:benchmarks": "nx build @lwc/perf-benchmarks",
+    "dev": "nx run-many --target=dev --all --parallel=999 --exclude=@lwc/perf-benchmarks,@lwc/perf-benchmarks-components,@lwc/integration-tests",
     "test": "jest --config ./scripts/jest/root.config.js",
     "test:debug": "node --inspect node_modules/.bin/jest --config ./scripts/jest/root.config.js --runInBand --watch",
     "test:ci": "yarn test --no-cache --coverage --runInBand",
-    "test:integration": "lerna exec --scope @lwc/integration-tests -- yarn sauce",
-    "test:performance": "lerna exec --scope @lwc/perf-benchmarks -- yarn test",
-    "release:publish:ci": "./scripts/release/publish.mjs",
-    "release:version": "./scripts/release/version.mjs"
+    "test:integration": "nx sauce @lwc/integration-tests",
```

Comment:
@nolanlawson I left `sauce` as an `nx` target but this is really only useful for local development.  As you mentioned before, in the CI we don't call this command at all. 

The benefit of leaving it as an `nx` target is that we can build the dependencies before running the integration tests.  

Let me know what you think, I don't have a strong opinion on whether we run this through `nx`.

Reply:
I'm still unsure about running any of our tests through `nx`. In this case, running `yarn test:integration` does not seem to actually skip the build steps for `@lwc/integration-tests`, presumably because we would need to tell nx that the `outputs` is `public`, not `dist`. We could add this configuration, but it seems low-value since it's not run in CI and the build takes only a few seconds on my machine.

For the perf tests, I feel similarly conflicted because I often prefer to run `yarn build:performance` manually, because I run it with different env variables that affect the output. NX does not know about these env variables or factor them in to the cache key.

That said, the alternative is to do `cd packages/<package> && yarn<cmd>`, which is pretty ugly. So let's keep it as-is.